dist/rpm/os-autoinst.spec

```diff
@@ -25,14 +25,9 @@ Group:          Development/Tools/Other
 Url:            https://github.com/os-autoinst/os-autoinst
 Source0:        %{name}-%{version}.tar.xz
 %{perl_requires}
-%if 0%{?suse_version} > 1500 || 0%{?sle_version} >= 150400
-# openSUSE Tumbleweed and Leap 15.4
 %define opencv_require pkgconfig(opencv4)
-%else
-%define opencv_require pkgconfig(opencv)
-%endif
 # exclude additional sub packages that would pull in a lot of extra dependencies on SLE
-%if 0%{?sle_version} && !0%{?is_opensuse}
+%if 0%{?suse_version} && !0%{?is_opensuse}
```

Comment:
I am not really sure about this condition. `sle_version` is not defined on Tumbleweed, `suse_version` is. Please double check if this condition does what we are expecting.

Reply:
```
[dheidler@midas ~]$ podman run --rm -it opensuse/leap:15.6 rpm --eval '%if 0%{?suse_version} && !0%{?is_opensuse}'
%if 01500 && !0
[dheidler@midas ~]$ podman run --rm -it opensuse/tumbleweed rpm --eval '%if 0%{?suse_version} && !0%{?is_opensuse}'
%if 01699 && !01

Reply:
Yeah, the condition is TRUE for Leap 15.6 which I am not sure if is the desired state (considering the comment mentions excluding some packages on SLE).

Reply:
this conversion is quite straight forward IMO. the only concern is the confusing `is_opensuse` which I am told _that %is_opensuse is not defined because that macro is defined via Project Config in the build service.` i guess that's why might return 0, even so it is represented as `1` in 
https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros